src/hooks/mode.hook.js

```diff
@@ -0,0 +1,37 @@
+import { useLayoutEffect, useState } from 'react'
+import localStorageService from '../utils/localStorageService'
+import { getUrlMode } from '../utils/helper'
+
+/**
+ * @hook
+ * Returns an appliction `mode` object
+ *
+ * isDemoMode = Not ready for production or testing (Missing UI or BE). shows all the code for staging + demo
+ *
+ * isStagingMode = When ready for testing but not for production. Specific for staging
+ *
+ * @returns {{isDemoMode: boolean, isStagingMode: boolean}}
+ *
+ * @example
+ *
+ * { isDemoMode: boolean, isStagingMode: boolean}
```

Comment:
example should contain the exact return value
`{ isDemoMode: false, isStagingMode: true }`